src/main/java/it/gov/pagopa/payment/service/payment/common/CommonPreAuthService.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.payment.service.payment.common;
+
+import it.gov.pagopa.payment.dto.AuthPaymentDTO;
+import it.gov.pagopa.payment.model.TransactionInProgress;
+
+public interface CommonPreAuthService {
+    TransactionInProgress relateUser(TransactionInProgress trx, String userId);
+    AuthPaymentDTO previewPayment(TransactionInProgress trx);
+    void auditLogUserRelate(TransactionInProgress trx);
```

Comment:
why auditLogUserRelate in the interface? it's supposed to be used outside the Service?